components/admin/lmod/SPECS/lmod.spec

```diff
@@ -151,6 +158,12 @@ else
    setenv MODULEPATH "%{OHPC_MODULES}"
 endif
 
+if ( ! $?MANPATH ) then
+    setenv MANPATH ":"
+endif
+# Initialize MANPATH if unset, then safely append Lmod's man directory using addto helper
+setenv MANPATH ` ${OHPC_ADMIN}/lmod/lmod/libexec/addto MANPATH ${OHPC_ADMIN}/lmod/lmod/share/man `
```

Comment:
I think there are unnecessary spaces after and before the backticks in this line. Please remove them. The spaces which are inside of the content between the backticks:
```
setenv MANPATH `${OHPC_ADMIN}/lmod/lmod/libexec/addto MANPATH ${OHPC_ADMIN}/lmod/lmod/share/man`
```